.prettierrc

```diff
@@ -2,9 +2,3 @@ useTabs: true
 tabWidth: 2
 printWidth: 100
 singleQuote: true
-trailingComma: es5
-bracketSpacing: true
-parenSpacing: true
-jsxBracketSameLine: false
-semi: true
-arrowParens: avoid
```

Comment:
These are default values + `parenSpacing` which was what the `wp-prettier` fork added.